test/spec/tasks/check_enabled_tests.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Mix.Tasks.CheckEnabledTests do
+  @moduledoc """
+  A task that checks which spec-tests are enabled or skipped. Useful to make sure that desired
+  tests are running.
+
+  It traverses vector configs, forks and runners hierarchically.
+
+  Run with MIX_ENV=test mix check_enabled_tests
+  The output will be in a reports/spec_test_report.exs file, that can be formatted using mix format.
+  """
+
+  require Logger
+
+  alias Spec.MetaUtils
+
+  def run(_args) do
```

Comment:
I haven't been able to run this from the CLI (tried with `mix check_enabled_tests`). I noticed it's missing the usual `use Mix.Task` and `@impl Mix.Task` tags, but even with that it doesn't work. Is it working for you?

Reply:
Fixed. You had to use MIX_ENV=test, but I added the env preference so you can skip the manual env config.